invenio_curations/assets/semantic-ui/js/invenio_curations/requests/RequestMetadataLayout.js

```diff
@@ -0,0 +1,265 @@
+import { i18next } from "@translations/invenio_requests/i18next";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { Image } from "react-invenio-forms";
+import Overridable from "react-overridable";
+import { Divider, Header, Icon, Message } from "semantic-ui-react";
+import { toRelativeTime } from "react-invenio-forms";
+import RequestStatus from "@js/invenio_requests/request/RequestStatus";
+import RequestTypeLabel from "@js/invenio_requests/request/RequestTypeLabel";
+import { connect } from "react-redux";
+import { connect as connectFormik } from "formik";
+
+const User = ({ user }) => (
+  <div className="flex">
+    <Image
+      src={user.links.avatar}
+      avatar
+      size="tiny"
+      className="mr-5"
+      ui={false}
+      rounded
+    />
+    <span>
+      {user.profile?.full_name ||
```

Comment:
could you please move that out of the jsx part. general rule, don't add logic to jsx if it is avoidable!